qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -39,6 +40,26 @@ def ls_dir(self, path, expectedLength, depth = None):
         self.assertEqual(len(data), expectedLength)
         return data
 
+    def setQuotas(self, bytes=None, files=None):
+        quotas = {
+            'max_bytes': bytes,
+            'max_files': files
+        }
+        self._post("/api/cephfs/{}/rm_snapshot".format(self.get_fs_id()), data=quotas,
+                   params={'path': '/quotas', 'name': 'test'})
+        self.assertStatus(200)
+
+    def assertQuotas(self, bytes, files):
+        data = self.ls_dir('/', 1)[0]
+        self.assertEqual(data['quotas']['max_bytes'], bytes)
+        self.assertEqual(data['quotas']['max_files'], files)
+
+    @contextmanager
+    def new_quota_dir(self):
```

Comment:
When this method would receive the quota, the quotas wouldn't had to be set twice and the `self.setQuotas(x)` line on different tests that use this method could be removed (L235, L240, L245).

Reply:
The reason to set it twice is to make sure that a change of one quota doesn't change the previous quota and overwrites it with 0. (Which happend with in the initial implementation.)